This patch is intended to allow the Makefile to be used on a debian system
that has the arduino-core package installed, but in the same manner as though
one had installed it locally -- so it now refers to the installed binaries etc.

Index: arduino-core-0017.0~svn802/hardware/cores/arduino/Makefile
===================================================================
--- arduino-core-0017.0~svn802.orig/hardware/cores/arduino/Makefile	2009-12-13 19:30:48.000000000 +0000
+++ arduino-core-0017.0~svn802/hardware/cores/arduino/Makefile	2009-12-13 19:48:05.000000000 +0000
@@ -4,6 +4,10 @@
 # This makefile allows you to build sketches from the command line
 # without the Arduino environment (or Java).
 #
+# Note, this Makefile has been edited by the Debian package maintainer, so
+# there's a reasonable chance it'll simply work without the need to edit it
+# but just in case, here are the instructions:
+#
 # Detailed instructions for using the makefile:
 #
 #  1. Copy this file into the folder with your sketch. There should be a
@@ -35,9 +39,9 @@
 # $Id$
 
 TARGET = $(notdir $(CURDIR))
-INSTALL_DIR = ../../..
-PORT = /dev/tty.usb*
-UPLOAD_RATE = 19200
+#INSTALL_DIR = ../../..
+PORT = /dev/ttyUSB*
+UPLOAD_RATE = 57600
 AVRDUDE_PROGRAMMER = stk500v1
 MCU = atmega168
 F_CPU = 16000000
@@ -45,8 +49,8 @@
 ############################################################################
 # Below here nothing should be changed...
 
-ARDUINO = $(INSTALL_DIR)/hardware/cores/arduino
-AVR_TOOLS_PATH = $(INSTALL_DIR)/hardware/tools/avr/bin
+ARDUINO = .
+AVR_TOOLS_PATH = /usr/bin
 SRC = $(ARDUINO)/pins_arduino.c $(ARDUINO)/wiring.c \
 $(ARDUINO)/wiring_analog.c $(ARDUINO)/wiring_digital.c \
 $(ARDUINO)/wiring_pulse.c $(ARDUINO)/wiring_shift.c \
